src/publisher/api_v2_views.py

```diff
@@ -35,23 +39,38 @@ def ctype(content_type_key, version=None):
     return "%s; version=%s" % (content_type, version)
 
 
-def ErrorResponse(code, title, detail=None):
-    body = {"title": title, "detail": detail}
-    if not detail:
-        del body["detail"]
-    return HttpResponse(
-        status=code, content_type="application/json", content=json.dumps(body)
-    )
+def response(data, code=200, content_type=None, headers=None):
+    content_type = content_type or "text/plain; charset=UTF-8"
```

Comment:
do I need to set the `charset`?

Reply:
turns out it's good practice. Adding it now is out of scope and will probably break a lot of tests.